catboost/R-package/R/catboost.R

```diff
@@ -396,6 +396,33 @@ print.catboost.Pool <- function(x, ...) {
 }
 
 
+#' @title Print basic information about model
+#' @description Displays the most general characteristics of a CatBoost model.
+#' @param x The model obtained as the result of training.
+#' @param ... Not used
+#' @return The same model that was passed as input.
+#' @export
+print.catboost.Model <- function(x, ...) {
+    cat(sprintf("CatBoost model (%d trees)\n", x$tree_count))
+    cat(sprintf("Evaluation metric: %s\n", catboost.get_plain_params(x)$eval_metric))
```

Comment:
Let's replace "Evaluation metric" with "Loss function" and `$eval_metric` with `$loss_function`.

Reply:
Done.